app/main/views/suppliers.py

```diff
@@ -439,6 +440,57 @@ def find_supplier_services():
     )
 
 
+@main.route(
+            '/suppliers/<string:supplier_name>/<int:supplier_code>/<string:domain_name>/'
+            '<int:supplier_domain_id>/<string:price>/update',
+            methods=['POST', "GET"])
+@login_required
+@role_required('admin')
+def update_supplier_category_price(supplier_name, supplier_code, domain_name, supplier_domain_id, price):
+    errorMsg = None
+    if request.args.get('form') is not None:
+        errorMsg = 'Please enter a valid price'
```

Comment:
If this is only doing a page load, we shouldn't have to set this error message here.

Reply:
This part is no longer in module